swtbot_test/src/testcase/TCExecute.java

```diff
@@ -86,7 +100,48 @@ public void TC_00_execute() throws ParseException {
 	}
 
 	@Test
-	public void TC_01_checkBuild() {
+	public void TC_01_checkConfigurationError() throws Exception {
+		int index = 0;
+		int length = bot.tree().visibleRowCount();
+		String currentProjectName ="";
+		ProjectModel projectModel = null;
+		boolean isThereConfigurationError = false;
+		while(true) {
+			currentProjectName = getCurrentProject(index);
+			projectModel = findProjectModelWithProjectName(currentProjectName);
+			bot.tree().getTreeItem(currentProjectName).select();
+			Utility.openSCFGEditor(projectModel, "Overview");
+			for (int i = 0; i < bot.table().rowCount(); i++) {
+				if (bot.table(0).getTableItem(i).getText(2).contains("configuration error")) {
+					System.out.println("Project "+ currentProjectName+" has configuration error");
+					isThereConfigurationError = true;
+				}
+				System.out.println(bot.table(0).getTableItem(i).getText(2));
+			}
+			index ++;
+			bot.closeAllEditors();
+			bot.tree().getTreeItem(projectModel.getProjectName() + " ["+ projectModel.getActiveBuildConfiguration() +"]").collapse();
+			if (index == length) {
+				break;
+			}
+		}
+		if (isThereConfigurationError) {
```

Comment:
E: There is no way to detect whichever apps have configuration error, only have result in general. Please improve it in the next commit.

Reply:
The new commit will write to a text file instead